test/parallel/test-timers-unref-reuse-no-exposed-list.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Does this need a license block now that https://github.com/nodejs/node/pull/10155 landed?